runtime/src/main/java/org/corfudb/util/serializer/DynamicProtobufSerializer.java

```diff
@@ -139,10 +139,8 @@ public DynamicProtobufSerializer(CorfuRuntime corfuRuntime) {
             String protoFileName = fileDescriptorProto.getPayload().getFileDescriptor().getName();
             // Since corfu_options is something within repo, the path gets truncated on insert.
             // However dynamicProtobufSerializer fails since the full path is needed.
-            if (protoFileName.equals("corfu_options.proto")) {
-                fdProtoMap.putIfAbsent(protoFileName, fileDescriptorProto.getPayload().getFileDescriptor());
-                // Until the truncating issue can be addressed, manually add both paths.
-                protoFileName = "corfudb/runtime/corfu_options.proto";
+            if (protoFileName.equals("corfudb/runtime/corfu_options.proto")) {
```

Comment:
why did the path change? was it not working before?

Reply:
I noticed  that the protoFileName changed on the setup. There is corfudb/runtime/corfu_options.proto but the corfu_options.proto is missing so I changed to this. Without this change it reports `corfu_options.proto is never seen in registry` error.